modules/benchmark/R/calc.benchmark.R

```diff
@@ -11,21 +11,29 @@
 
 calc.benchmark <- function(settings, bety) {
   
-  library(RPostgreSQL)
-  library(data.table)
+  # dplyr functions
+  tbl     <- dplyr::tbl
+  filter  <- dplyr::filter
+  rename  <- dplyr::rename
+  collect <- dplyr::collect
+  select  <- dplyr::select
   
```

Comment:
I agree except for the places where a particular function is used a large number of times